ios/Podfile

```diff
@@ -12,6 +12,8 @@ target 'growupmobile' do
     :hermes_enabled => false
   )
 
+  pod 'react-native-pager-view', :path => '../node_modules/react-native-pager-view'
```

Comment:
Do we really need this manual linking?
I assume we already have this library installed and being used